navigator/frontend/src/ui-core/src/session/UI.tsx

```diff
@@ -85,15 +102,32 @@ export default class Component<A extends Action>
     let errorEl = null;
     if (failure === 'invalid-credentials') {
       errorEl = (
-        <p>Sorry, those credentials are invalid</p>
+        <ErrorMessage>
+          <div>You don't have the necessary authorization to access</div>
```

Comment:
```suggestion
          <div>You don't have the necessary authorization to access the ledger</div>
```

Sounds like there is an object missing at the end?

Reply:
This line was on purpose but ok.